tools/apidocgen.py

```diff
@@ -0,0 +1,176 @@
+"""
+Generate RST documentation for an ubuntu-pro-client api endpoint.
+
+Usage: env PYTHONPATH=. python3 tools/apidocgen.py $ENDPOINT_NAME
+"""
+
+import sys
+import textwrap
+from importlib import import_module
+
+from uaclient import data_types
+
+NO_ARGS = "- This endpoint takes no arguments."
+NO_EXCEPTIONS = "- No exceptions raised by this endpoint."
```

Comment:
now may be the time to fix this kinda incorrect information
this is more like "no PARTICULAR exception expected to be raised here"

because see - all endpoints may spit UbuntuProErrors if things go wrong?

Reply:
Yes good catch!